src/http/handler/HistoryHandler.java

```diff
@@ -0,0 +1,36 @@
+package http.handler;
+
+import com.google.gson.Gson;
+import com.sun.net.httpserver.HttpExchange;
+import http.HttpTaskServer;
+import managers.TaskManager;
+import tasks.Task;
+
+import java.io.IOException;
+import java.util.List;
+
+public class HistoryHandler extends BaseHttpHandler {
+    private final TaskManager manager;
+    private final Gson gson = HttpTaskServer.getGson();
```

Comment:
Можно вынести в `BaseHttpHandler`, чтобы не дублировать во всех остальных handler'ах

(You can move it to `BaseHttpHandler` to avoid duplicating it in all other handlers)